src/NServiceBus.Persistence.CosmosDB.Tests/Outbox/LogicalOutboxBehaviorTests.cs

```diff
@@ -160,6 +160,12 @@ public override Task<ResponseMessage> ReadItemStreamAsync(string id, PartitionKe
 
         public override TransactionalBatch CreateTransactionalBatch(PartitionKey partitionKey) => throw new NotImplementedException();
 
+        public override FeedIterator<T> GetChangeFeedIterator<T>(ChangeFeedStartFrom changeFeedStartFrom, ChangeFeedMode changeFeedMode, ChangeFeedRequestOptions changeFeedRequestOptions = null) => throw new NotImplementedException();
+
+        public override FeedIterator GetChangeFeedStreamIterator(ChangeFeedStartFrom changeFeedStartFrom, ChangeFeedMode changeFeedMode, ChangeFeedRequestOptions changeFeedRequestOptions = null) => throw new NotImplementedException();
+
+        public override Task<IReadOnlyList<FeedRange>> GetFeedRangesAsync(CancellationToken cancellationToken = default) => throw new NotImplementedException();
+
```

Comment:
Interesting. So a clear breaking change in a minor?

Reply:
Yeah guess so. Maybe they don’t consider the abstract class to really be part of the public API, but 🤷‍♂️